css/style.css

```diff
@@ -331,45 +338,44 @@ textarea {
   background-color: #fff;
   border: solid 1px grey;
   border-radius: 4px;
-  box-sizing: border-box;
-  margin-bottom: 8px;
-  padding: 0 8px;
+  margin-bottom: 2.5vw;
+  padding: 0 2.5vw;
   position: relative;
-  width: 288px;
-  height: 52px;
+  height: 16.25vw;
 }
 
 .float-textarea {
-  height: 112px;
+  height: 35vw;
   background-color: #fff;
   border: solid 1px grey;
   border-radius: 4px;
-  box-sizing: border-box;
-  margin-bottom: 8px;
-  padding: 0 8px;
+  margin-bottom: 2.5vw;
+  padding: 0 2.5vw;
   position: relative;
-  width: 288px;
+}
+
+.float-textarea span {
+  visibility: hidden;
 }
 
 /* Contact form end */
 
 footer {
   background-color: #1a2236;
-  width: 320px;
-  height: 150px;
+  width: 100vw;
```

Comment:
- [ ] Please notice that using the `vw` unit in the `width` property is causing an overflow on the website. Kindly keep in mind that [`vw`](https://stackoverflow.com/questions/25225682/difference-between-width100-and-width100vw) stands for "viewport width" and this means that this element is going to take all the available space on the screen including the document margin. Please use the `%` unit instead or just delete the line (it won't make a difference) in order to fix the design issue.

> Please consider this note in the future and on the other stylings you made in this project.

Reply:
Thank you, this helped me a lot! ✌️

Reply:
fixed 👍